src/fprime_gds/common/data_types/ch_data.py

```diff
@@ -37,6 +38,7 @@ def __init__(self, ch_val_obj, ch_time, ch_temp):
         self.time = ch_time
         self.template = ch_temp
         self.pkt = None
+        self.display_text = self._compute_display_text(ch_val_obj, ch_temp)
```

Comment:
I thought we discussed making this a function that is called when needed, not forcing it to be "right" at initialization.

Reply:
i.e. `_compute_display_text` -> `get_display_text` so that it is called often.  By making it a class member, it will not be recalculated in the case something changes.

Reply:
_Can_ things change? I have no issue reverting back, but I'm wondering at what point the value object would change?

Reply:
We decided it was invalid for things to change because:
1. The `val_obj` is set on construction
2. The presence of getters without setters implies an interface expecting no changes
3. The only known changer of these objects was refactored out